csi/controller_server.go

```diff
@@ -268,7 +273,8 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	}, nil
 }
 
-func (cs *ControllerServer) getBackupVolume(volumeName string) (*longhornclient.BackupVolume, error) {
+func (cs *ControllerServer) getBackupVolume(volumeName string) ([]*longhornclient.BackupVolume, error) {
```

Comment:
Even with the return signature changed, I think the function name should also change to `getBackupVolumes`

Reply:
Agree with @james-munson. The function name is confusing.

Reply:
I agree. Fixed. Thank you